services/QuillLMS/app/controllers/invitations_controller.rb

```diff
@@ -59,14 +59,14 @@ def destroy_pending_invitations_from_specific_inviter
   end
 
   private def validate_email_format
-    return if @invitee_email =~ /.+@.+\..+/i
+    return if @invitee_email =~ User::VALID_EMAIL_REGEX
 
     raise StandardError, "Please make sure you've entered a valid email."
   end
 
   private def set_classroom_ids_and_inviteee_email
     @classroom_ids = params[:classroom_ids]
-    @invitee_email = params[:invitee_email]
+    @invitee_email = params[:invitee_email]&.gsub(/\s/, '') #strip whitespace
```

Comment:
Should we just use `.strip` here and rely on the validation regex to catch instances where there is whitespace in the email address itself?  Probably doesn't matter either way.

Reply:
I don't think it's a big deal either way. Looking at the [failed invites](https://data.quill.org/question/958-invalid-invites), there are several that would be valid if we stripped the space and seem like obvious typos (e.g. `email@gmail. com`). Given emails can't have whitespace in the middle, I think it's so it's ok to attempt to fix that for the user, then validate, rather than sending it back to them to fix (I could be convinced otherwise).

Reply:
That seems reasonable to me.  I don't object to us fixing something for users rather than making them do it.